frontend/apps/reader/modules/readercoptlistener.lua

```diff
@@ -18,10 +26,179 @@ function ReaderCoptListener:onReadSettings(config)
+                    })
+                end,
+                separator = true,
+            },
+            {
+                text = _("Title"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.title == 1
+                end,
+                callback = function()
+                    self.title = 1 - self.title
+                    self:setAndSave("window.status.title", "cre_header_title", self.title)
+                end,
+            },
+            {
+                text = _("Clock"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.clock == 1
+                end,
+                callback = function()
+                    self.clock = 1 - self.clock
+                    self:setAndSave("window.status.clock", "cre_header_clock", self.clock)
+                end,
+            },
+            {
+                text = _("Page number"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.page_number == 1
+                end,
+                callback = function()
+                    self.page_number = 1 - self.page_number
+                    self:setAndSave("window.status.pos.page.number", "cre_header_page_number", self.page_number)
+                end,
+            },
+            {
+                text = _("Page count"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.page_count == 1
+                end,
+                callback = function()
+                    self.page_count = 1 - self.page_count
+                    self:setAndSave("window.status.pos.page.count", "cre_header_page_count", self.page_count)
+                end,
+            },
+            {
+                text = _("Reading percent"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.reading_percent == 1
+                end,
+                callback = function()
+                    self.reading_percent = 1 - self.reading_percent
+                    self:setAndSave("window.status.pos.percent", "cre_header_reading_percent", self.reading_percent)
+                end,
+            },
+            {
+                text = _("Battery"),
+                keep_menu_open = true,
+                checked_func = function()
+                    return self.battery == 1
+                end,
+                callback = function()
+                    self.battery = 1 - self.battery
+                    self:setAndSave("window.status.battery", "cre_header_battery", self.battery)
+                end,
+            },
+            {
+                text = _("Battery percent"),
+                keep_menu_open = true,
+                enabled_func = function()
+                    return self.battery == 1
```

Comment:
Is `window.status.battery.percent` really dependant on `window.status.battery` being enabled?

Reply:
Yes. Battery percent is only shown if battery is enabled